homeassistant/components/knx/__init__.py

```diff
@@ -208,13 +219,21 @@ def connection_config_tunneling(self):
         gateway_ip = self.config[DOMAIN][CONF_KNX_TUNNELING].get(CONF_HOST)
```

Comment:
Changing this as well because it config key is mandatory and therefore required.
```suggestion
        gateway_ip = self.config[DOMAIN][CONF_KNX_TUNNELING][CONF_HOST]
```